test/integration/price-import.spec.coffee

```diff
@@ -87,16 +87,24 @@ describe 'Price Importer integration tests', ->
     prices = [
       {
         sku: 'sku1'
-        value:
-          centAmount: 9999
-          currencyCode: 'EUR'
+        prices: [
+          {
+            value:
+              centAmount: 9999
+              currencyCode: 'EUR'
+          }
+        ]
```

Comment:
thats how.. stripped down sample products and their variants look like: 
http://pastie.org/private/6ufycyax8herfxlur0wvig
in the prices file at the time of transformation all I have is sku of each variant.